apps/server/lib/google-drive/src/resources/teamdrives.ts

```
@@ -0,0 +1,1361 @@
+export namespace TeamdriveDeleteParams {
+	export interface _ {
+		/**
+		 * V1 error format.
+		 */
+		xgafv?: "1" | "2";
+	}
+}
+
+export declare namespace Teamdrives {
+	export {
+		type TeamDrive as TeamDrive,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)